<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Apples-MacBook-Air.local" failures="4" tests="4" name="Test(failed)(failed)" time="47.794" errors="0" timestamp="2022-03-07T23:07:16 IST">
  <testcase classname="com.xyzbanking.testCase.TC_LoginDDT_002" name="loginDDT" time="7.825">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.xyzbanking.testCase.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.xyzbanking.testCase.TC_LoginDDT_002" name="loginDDT" time="7.344">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.xyzbanking.testCase.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.xyzbanking.testCase.TC_LoginDDT_002" name="loginDDT" time="7.246">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.xyzbanking.testCase.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.xyzbanking.testCase.TC_LoginTest_001" name="loginTest" time="2.522">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at com.xyzbanking.testCase.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Test(failed)(failed) -->
